flink-runtime/src/main/java/org/apache/flink/runtime/entrypoint/ClusterEntrypoint.java

```diff
@@ -263,6 +264,7 @@ private void runCluster(Configuration configuration, PluginManager pluginManager
                             new RpcMetricQueryServiceRetriever(
                                     metricRegistry.getMetricQueryServiceRpcService()),
                             this);
+            configuration.setString(RestOptions.BIND_PORT, clusterComponent.getRestPort());
```

Comment:
I believe we do not need this line of change after introducing the `webInterfaceUrl` in `KubernetesResourceManagerDriver`.